docs/contributing/pull_requests.md

```diff
@@ -31,3 +31,8 @@ Neurobagel PR reviews may use the following emoji signifiers:
 :cherries:: Some optional/suggested changes that could be nice to have but are not required to merge
 
 If (required) changes are requested, the PR author should re-request a review from the reviewer once the comments have been addressed.
+
+### When your pull request is approved
+For PRs opened by community contributors, the reviewing Neurobagel maintainer is responsible for merging the PR.
```

Comment:
```suggestion
If you are **not a core maintainer**: For PRs opened by community contributors, the reviewing Neurobagel maintainer is responsible for merging the PR.
```

Reply:
Should we say "Neurobagel developer" instead of "Core maintainer" to differentiate folks who have write access to the repos but aren't you/me/Arman?

Reply:
yeah, great idea. Actually you might even say that part: if you have write access (e.g. you are a neurobagel maintainer)